CHANGELOG.asciidoc

```diff
@@ -68,6 +68,9 @@ Notes:
 [float]
 ===== Chores
 
+* The config option <<filter-http-headers, filterHttpHeaders>> is now *deprecated*. It will be
+  removed in a future major version (after an 18 month deprecation period). ({pull}3333[#3333])
```

Comment:
```suggestion
  removed in a future major version. ({pull}3333[#3333])
```

I'm not sure we need to mention an 18 month deprecation period. I expect we'll have a 4.x release before 18 months as well. Did you get the 18 months from some policy doc that I am forgetting?

Reply:
Just looked a previous entries of this file and found deprecation ones came with that comment so I assumed it was the norm.